test/integration/features/basic.feature

```diff
@@ -192,6 +192,24 @@ Feature: Basic
      When executing "minishift stop" succeeds
      Then Minishift should have state "Stopped"
 
+  Scenario: Stopping non existing VM
```

Comment:
that does not work. After stopping the VM is still there, so the Given clause will fail.